server/modules/category/repository/categoryRepository.js

```diff
@@ -194,6 +206,12 @@ const updateCategoryValidation = async (surveyId, categoryUuid, validation, clie
     [validation, categoryUuid]
   )
 
+const markCategoriesPublished = async (surveyId, client = db) =>
```

Comment:
I would call this method ```markCategoriesPublishedBySurveyId``` what do you think?

Reply:
Is there any other markCategoriesPublished we should distinguish from?

Reply:
just to be consistent with other methods in the same module. whatever works in the end, your call

Reply:
ok